Source/WebKit/WebProcess/Extensions/WebExtensionContextProxy.h

```diff
@@ -124,9 +125,9 @@ class WebExtensionContextProxy final : public RefCounted<WebExtensionContextProx
     Vector<Ref<WebPage>> tabPages(std::optional<WebExtensionTabIdentifier> = std::nullopt, std::optional<WebExtensionWindowIdentifier> = std::nullopt) const;
     void addTabPage(WebPage&, std::optional<WebExtensionTabIdentifier>, std::optional<WebExtensionWindowIdentifier>);
 
-    void enumerateFramesAndNamespaceObjects(const Function<void(WebFrame&, WebExtensionAPINamespace&)>&, WebCore::DOMWrapperWorld& = mainWorld());
+    void enumerateFramesAndNamespaceObjects(const Function<void(WebFrame&, WebExtensionAPINamespace&)>&, Ref<WebCore::DOMWrapperWorld> = protectedMainWorld());
 
-    void enumerateNamespaceObjects(const Function<void(WebExtensionAPINamespace&)>& function, WebCore::DOMWrapperWorld& = mainWorld())
+    void enumerateNamespaceObjects(const Function<void(WebExtensionAPINamespace&)>& function, Ref<WebCore::DOMWrapperWorld> = protectedMainWorld())
```

Comment:
Sorry, I cannot review here. I don't get the usage of unnamed variable with default value with concrete implementation. Anyway, what I can say is that there's no need to change the signature of `enumerateNamespaceObjects()` as long as the passed argument is protected.

Reply:
Offline discussion with Ryosuke, it seems that we can simply remove this parameter. It's not used in the function itself and no call sites pass it in.